### PR TITLE
add docs for rate limits

### PR DESCRIPTION
**Pull request by bpal938**
This adds new docs to explain our rate limits at a high level.

Test Plan: 
- Ensure the new docs can be seen at https://docs.centrapay.com/api/rate-limits

image
image

### CODE REVIEW

File: src/content/api/http-status-codes.mdoc [hunk 1]
```diff
@@ -98,7 +98,7 @@ the URL or HTTP method. Please check against examples in our documentation.
 
 ## 429 Too Many Requests
 
-Centrapay API rate limits have been exceeded.
+Centrapay API rate limits have been exceeded. See [Rate Limits](/api/rate-limits) for more details
```
Comment (laurencemcknight) [inline src/content/api/http-status-codes.mdoc:101]:
```suggestion
Centrapay API rate limits have been exceeded. See [Rate Limits](/api/rate-limits) for more details.
```

File: src/content/api/rate-limits.mdoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+At Centrapay rate limits are defined by **Requests Per Minute (rpm).** 
+
+The general rate limit is 200rpm, however for certain APIs we may apply a specific lower or higher rate limit in order to match the expected usage of the endpoint.
+
+Rate limits for authenticated endpoints are applied to the account making the call. This means if you are using a `X-Centrapay-Account` header, the rate limits will still be consumed for the account that the jwt or api key belongs to, not the targeted account.
+
+If you require higher rate limits for your use case, please contact support to request an increase.
+
+## Preventing Rate Limits
```
Comment (laurencemcknight) [inline src/content/api/rate-limits.mdoc:18]:
```suggestion
## Preventing Rate Limiting
```

**Review by laurencemcknight — APPROVED** [review]
You will need to fix the integration test but other than that LGTM.

File: src/content/api/http-status-codes.mdoc [hunk 3]
```diff
@@ -98,7 +98,7 @@ the URL or HTTP method. Please check against examples in our documentation.
 
 ## 429 Too Many Requests
 
-Centrapay API rate limits have been exceeded.
+Centrapay API rate limits have been exceeded. See [Rate Limits](/api/rate-limits) for more details.
```
Comment (Ray2126) [inline src/content/api/http-status-codes.mdoc:101]: Suggestion: Perhaps we should remove the whole "debugging" section and replace with a link to the new page i.e. move the new sentence here in place of the debugging section.

File: src/content/api/rate-limits.mdoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+---
+In order to maintain availability to our services by protecting against rapid requests, Centrapay uses Rate Limits to restrict how many requests can be made within a given time period.
+
+At Centrapay rate limits are defined by **Requests Per Minute (rpm).** 
+
+The general rate limit is 200rpm, however for certain APIs we may apply a specific lower or higher rate limit in order to match the expected usage of the endpoint.
+
+Rate limits for authenticated endpoints are applied to the account making the call. This means if you are using a `X-Centrapay-Account` header, the rate limits will still be consumed for the account that the jwt or api key belongs to, not the targeted account.
+
+If you require higher rate limits for your use case, please contact support to request an increase.
```
Comment (Ray2126) [inline src/content/api/rate-limits.mdoc:16]: Change: Everywhere that we mention to contact support, we should specify the actual email address - integrations@centrapay.com

**Review by Ray2126 — APPROVED** [review]

**Pull request merged**